### ISSUE TITLE
Can't require descriptions on TS Property Signatures

### ISSUE THREAD

**Issue by snowystinger** [gajus/eslint-plugin-jsdoc#582]
## Expected behavior

```js
{
      code: `
          interface quux {
            checked?: boolean
          }
      `,
      errors: [
        {
          message: 'Missing JSDoc @description declaration.',
        },
      ],
      options: [
        {
          contexts: [
            'TSPropertySignature',
          ],
          descriptionStyle: 'tag',
        },
      ],
      parser: require.resolve('@typescript-eslint/parser'),
    },
```

## Actual behavior

This passes even though `checked` doesn't have a description.

## ESLint sample

```js
// Format code using Markdown code blocks
```

## Environment

* Node version: 
* ESLint version 
* `eslint-plugin-jsdoc` version:

**Comment by snowystinger** [comment]
Ok, so now I understand require-jsdoc rule, however, should this still fail if there is no jsdoc at all?

**Comment by brettz9** [comment]
Yes, the rules are meant to be atomic, so `require-jsdoc` is required for reporting of missing jsdocs. The other rules report when there already is a doc block (albeit including potentially empty ones such as just `/** */`)

**Comment by brettz9** [comment]
Will close then as I think that should address, though feel free to comment further as needed.
